lib/rules/prefer-query-by-disappearance.ts

```diff
@@ -59,13 +67,20 @@ export default createTestingLibraryRule<Options, MessageIds>({
 				return false;
 			}
 
-			return (
+			if (
 				helpers.isGetQueryVariant(argumentProperty) ||
 				helpers.isFindQueryVariant(argumentProperty)
-			);
+			) {
+				context.report({
+					node: argumentProperty,
+					messageId: 'preferQueryByDisappearance',
+				});
```

Comment:
I find it quite confusing that the function is just checking if the expression is reportable, but if so it reports it in here. Could you rename the function, so it's called something like `reportExpression`, and then indicate the behavior in the jsdoc? Something simple, like _"Checks if node is reportable and report it if so. Returns boolean indicating if was reported"_.

Reply:
Good point. I re-worded both a bit and to the direction you suggested. Is it any better now?

Reply:
Looks great now!